library/Zend/Cloud/QueueService/Adapter/WindowsAzure.php

```diff
@@ -87,10 +87,10 @@ public function __construct($options = array())
             $host = $options[self::HOST];
         }
         if (! isset($options[self::ACCOUNT_NAME])) {
-            throw new Zend_Cloud_Storage_Exception('No Windows Azure account name provided.');
```

Comment:
This exception doesn't exist, so changed to one that's used elsewhere in this method.